examples/AspNetCore/Startup.cs

```diff
@@ -23,15 +23,19 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 using Microsoft.OpenApi.Models;
+using OpenTelemetry;
 using OpenTelemetry.Exporter;
 using OpenTelemetry.Instrumentation.AspNetCore;
+using OpenTelemetry.Metrics;
 using OpenTelemetry.Resources;
 using OpenTelemetry.Trace;
 
 namespace Examples.AspNetCore
 {
     public class Startup
     {
+        private MeterProvider metricsProvider;
```

Comment:
curious - why is called metricsprovider vs. meterprovider?

Reply:
Good catch. Case of 🧠 thinking one thing and ✋ typing another. 😄